tests/translate/translate_map1_ppm_2d.py

```diff
@@ -4,6 +4,12 @@
 from fv3core.testing import TranslateFortranData2Py, TranslateGrid
 
 
+def pad_field_in_j(field, nj):
+    outfield = np.tile(field[:, 0, :], [nj, 1, 1]).transpose(1, 0, 2)
+    np.testing.assert_array_equal(outfield[:, 0, :], field[:, 0, :])
+    return outfield
```

Comment:
This is a different method than used in the remap_profile_2d to achieve the same thing, should they share the same approach for less confusion next time we revisit these? Or are they different?

Reply:
Good question! I initially wrote `pad_field_in_j` in translate_remap_profile_2d.py, but it seems it wasn't necessary there or in the other remapping classes. I agree that we should be using the same method for both, but I was nervous to implement that without potentially breaking more things

Reply:
I think it's fine to leave as-is for now